### ISSUE TITLE
Can't develop locally with docker

### ISSUE THREAD

**Issue by kakascx** [Kong/docs.konghq.com#1364]
- `docs.konghq.com` url(s):

### Summary

I produced a custom plugin and attempt to make a contribution to Kong 
After add my index.md, I attempt to build the kong website by developing locally with docker.But there's some mistake when building it. It report `jekyll_1  | npm ERR! Error: EACCES: permission denied, unlink '/usr/bin/yarnpkg'`

### Steps To Reproduce

1.I add my own index.md and logo into the right dictionary
2.Execute the command `make develop` under the dictionary docs.konghq.com
3.It reported two errors `jekyll_1  | npm ERR! Error: EACCES: permission denied, unlink '/usr/bin/yarnpkg'` and `jekyll_1  | npm ERR! Error: EACCES: permission denied, unlink '/usr/bin/gulp'`

### Additional Details

The error logs 
`jekyll_1  | npm ERR! path /usr/bin/yarnpkg
jekyll_1  | npm ERR! code EACCES
jekyll_1  | npm ERR! errno -13
jekyll_1  | npm ERR! syscall unlink
jekyll_1  | npm ERR! Error: EACCES: permission denied, unlink '/usr/bin/yarnpkg'
jekyll_1  | npm ERR!  { [Error: EACCES: permission denied, unlink '/usr/bin/yarnpkg']
jekyll_1  | npm ERR!   cause:
jekyll_1  | npm ERR!    { Error: EACCES: permission denied, unlink '/usr/bin/yarnpkg'
jekyll_1  | npm ERR!      errno: -13,
jekyll_1  | npm ERR!      code: 'EACCES',
jekyll_1  | npm ERR!      syscall: 'unlink',
jekyll_1  | npm ERR!      path: '/usr/bin/yarnpkg' },
jekyll_1  | npm ERR!   stack:
jekyll_1  | npm ERR!    'Error: EACCES: permission denied, unlink \'/usr/bin/yarnpkg\'',
jekyll_1  | npm ERR!   errno: -13,
jekyll_1  | npm ERR!   code: 'EACCES',
jekyll_1  | npm ERR!   syscall: 'unlink',
jekyll_1  | npm ERR!   path: '/usr/bin/yarnpkg' }
jekyll_1  | npm ERR! 
jekyll_1  | npm ERR! The operation was rejected by your operating system.
jekyll_1  | npm ERR! It is likely you do not have the permissions to access this file as the current user
jekyll_1  | npm ERR! 
jekyll_1  | npm ERR! If you believe this might be a permissions issue, please double-check the
jekyll_1  | npm ERR! permissions of the file and its containing directories, or try running
jekyll_1  | npm ERR! the command again as root/Administrator (though this is not recommended).
jekyll_1  | 
jekyll_1  | npm ERR! A complete log of this run can be found in:
jekyll_1  | npm ERR!     /home/jekyll/.npm/_logs/2019-06-26T08_38_51_088Z-debug.log
`

`jekyll_1  | npm ERR! path /usr/bin/gulp
jekyll_1  | npm ERR! code EACCES
jekyll_1  | npm ERR! errno -13
jekyll_1  | npm ERR! syscall unlink
jekyll_1  | npm ERR! Error: EACCES: permission denied, unlink '/usr/bin/gulp'
jekyll_1  | npm ERR!  { [Error: EACCES: permission denied, unlink '/usr/bin/gulp']
jekyll_1  | npm ERR!   cause:
jekyll_1  | npm ERR!    { Error: EACCES: permission denied, unlink '/usr/bin/gulp'
jekyll_1  | npm ERR!      errno: -13,
jekyll_1  | npm ERR!      code: 'EACCES',
jekyll_1  | npm ERR!      syscall: 'unlink',
jekyll_1  | npm ERR!      path: '/usr/bin/gulp' },
jekyll_1  | npm ERR!   stack: 'Error: EACCES: permission denied, unlink \'/usr/bin/gulp\'',
jekyll_1  | npm ERR!   errno: -13,
jekyll_1  | npm ERR!   code: 'EACCES',
jekyll_1  | npm ERR!   syscall: 'unlink',
jekyll_1  | npm ERR!   path: '/usr/bin/gulp' }
jekyll_1  | npm ERR! 
jekyll_1  | npm ERR! The operation was rejected by your operating system.
jekyll_1  | npm ERR! It is likely you do not have the permissions to access this file as the current user
jekyll_1  | npm ERR! 
jekyll_1  | npm ERR! If you believe this might be a permissions issue, please double-check the
jekyll_1  | npm ERR! permissions of the file and its containing directories, or try running
jekyll_1  | npm ERR! the command again as root/Administrator (though this is not recommended).
`

**Comment by hutchic** [comment]
Those errors should be ignorable in my experience.

![Capture](https://user-images.githubusercontent.com/697188/60188039-a22e3200-97fc-11e9-89e1-a333d86c46c3.PNG)

here's a screenshot of that section from my run of `make develop` whereby it gets the error but continues on until it eventually

```
jekyll_1  | [09:25:08] Finished 'default' after 1.43 min
jekyll_1  | [ ▶ ] Access URLs:
jekyll_1  |  -----------------------------------
jekyll_1  |        Local: http://localhost:3000
jekyll_1  |     External: http://172.19.0.2:3000
jekyll_1  |  -----------------------------------
jekyll_1  |           UI: http://localhost:3001
jekyll_1  |  UI External: http://localhost:3001
jekyll_1  |  -----------------------------------
jekyll_1  | [ ▶ ] Serving files from: dist
```

**Comment by kakascx** [comment]
I have been solved this problem and built successful. There was two problem when I attempted to build it and  I think it's maybe helpful to others.
* The problem of permission 
It's the  problem of file owner and group. I work with the root right all the time, but  the file owner and group in docker is my username. Run `chown -R <username>:<username> docs.konghq.com/` and then it works. 
* The ram of the docker 
After solve the permission problem, there was another problem that the docker container exited with code 137. It's the problem caused by the docker ram constraint. I work under ubuntu in the VMWare and allocate 1G ram. Because of the resource constraint of docker, the building process exited with cod 137. After increase the ram, I solved the problem and built successful. I think it maybe happen in high probability  who works in virtual machine which has a low resource.
